### PR TITLE
Fix all linting

### PR DESCRIPTION
**Pull request by ctperry0301**
A looooooot of files were changed, Although I tested each aspect thoroughly, it should betested further before approval. 

check eslintrc.js file and make sure that were okay with the ignored rules

### CODE REVIEW

**Comment by ctperry0301** [comment]
Also, this should probably be merged in first so that other PR's wont have linting issues

File: components/CreateCycle/ExerciesView.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import { useSelector } from 'react-redux';
+import { View } from 'react-native';
+import { FlatList } from 'react-native-gesture-handler';
+import styled from 'styled-components/native';
+import React from 'react';
+import _ from 'lodash';
```
Comment (jpoist97) [inline components/CreateCycle/ExerciesView.jsx:6]: Thank you for moving this out

File: .eslintrc.js [hunk 2]
```diff
@@ -18,5 +18,11 @@ module.exports = {
     'react',
   ],
   rules: {
+    'max-len': 'off', // Kind of useless, for code style
+    'radix': 'off', // Useless, we wont be dealing with hex values
+    'global-require': 'off', // this one is fine because were requiring image filepaths,  
+    'react/forbid-prop-types': 'off', // this ideally would be fixed, but were in too deep 
+    'react/prop-types': 'off', // this ideally would be fixed, but were in too deep 
+    'no-unused-expressions': ["error", { "allowTernary": true }] // Although ternary isn't optimized, it is still useful
```
Comment (jpoist97) [inline .eslintrc.js]: I think we should leave this on. I took a look at all of the places where this rule gets violated and those should all be changed to if/else statements.

**Review by jpoist97 — COMMENTED** [review]
Thanks for doing all of this Cole, I just have one comment

File: .eslintrc.js (hunk 2 above)
Comment (ctperry0301) [reply .eslintrc.js]: Yeah, good call. Should be fixed now

**Review by jpoist97 — APPROVED** [review]
Looks good to me. Thanks Cole you the homie for this one

**Pull request merged**